public/app/plugins/panel/flamegraph/components/FlameGraph/colors.ts

```diff
@@ -61,6 +64,41 @@ export function getBarColorByPackage(label: string, theme: GrafanaTheme2) {
   return packageColor;
 }
 
+// green to red
+const diffDefaultColors = ['rgb(0, 170, 0)', 'rgb(148, 142, 142)', 'rgb(200, 0, 0)'];
+export const diffDefaultGradient = `linear-gradient(90deg, ${diffDefaultColors[0]} 0%, ${diffDefaultColors[1]} 50%, ${diffDefaultColors[2]} 100%)`;
+const diffColorBlindColors = ['rgb(26, 133, 255)', 'rgb(148, 142, 142)', 'rgb(220, 50, 32)'];
+export const diffColorBlindGradient = `linear-gradient(90deg, ${diffColorBlindColors[0]} 0%, ${diffColorBlindColors[1]} 50%, ${diffColorBlindColors[2]} 100%)`;
```

Comment:
So I know we're basically reusing colors from Pyroscope, but I'm wondering if we could incorporate more of the Grafana color palette

Reply:
I was told that pyro flamegraph actually used colors from grafana I am just not sure how to map them now, maybe we changed them in the meantime. I want to keep this for now at least so that when we swap the flamegraph in pyro app we can easily compare and tweak the colors afterwards